molgenis-api-tests/src/test/java/org/molgenis/api/tests/rest/v1/RestControllerIT.java

```diff
@@ -211,11 +211,10 @@ public void testRetrieveSystemEntityTypeNotAllowed()
 		// @formatter:off
 		given().log()
 			   .all()
-			   .header(X_MOLGENIS_TOKEN, this.testUserToken)
 			   .when()
 			   .get(PATH + "sys_App/meta")
 			   .then()
-			   .statusCode(FORBIDDEN)
+			   .statusCode(UNAUTHORIZED)
 			   .body("errors.code[0]", equalTo("DS04"))
 			   .body("errors.message[0]",
 					   equalTo("No 'Read metadata' permission on entity type 'App' with id 'sys_App'."));
```

Comment:
Running this one anonymously, because the test user gets read meta permissions on sys_App.